.github/workflows/goreleaser.yml

```diff
@@ -32,6 +32,9 @@ jobs:
     - name: Set LDFLAGS
       run: echo LDFLAGS="$(make ldflags)" >> $GITHUB_ENV
 
+    - name: Install syft binary executable
+      uses: anchore/sbom-action/download-syft@v0.18.0
```

Comment:
Is this enough to get the `syft` command into the `$PATH` that will be used by goreleaser?

It would be great to see a successful test.